content/devops/devsecops/2-commit.md

```diff
@@ -0,0 +1,53 @@
+
+Developer teams should ensure that they know what components are included in their applications, be sure that secure and up-to-date versions are downloaded from the known sources, and there's a process for keeping them up to date. It's also important to address package feeds security.
+
+To check the security on any dependencies you may use in your project:
+
+- [Dependabot](https://github.com/dependabot/dependabot-core#dependabot) - dependency review and alerts in GitHub. [Demo](https://github.com/dependabot/demo)
+- [OWASP Dependency-check project](https://owasp.org/www-project-dependency-check/)
+- [Securing software supply chain](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities) in GitHub.
+
+## Securing the CI/CD workflow
+
+Securing your CI/CD workflow makes a lot of sense, since it gives a point of access to the different environments you deploy to. This implies that you need to take extra care of into defining who has access to your repositories, pipelines and workflows and how each of your automatic processes are governed.
+
+For overall governance of your CI/CD processes, take a look at the [DevOps Governance session](../patterns-and-practices/readme.md) we host in [FTA Live](https://fasttrack.azure.com/live/category/DevOps).
+
+<img src="https://raw.githubusercontent.com/Azure/devops-governance/main/images/e2e-governance-overview.svg" width="640" alt="Azure Architect Center - E2E Governance">
+
+This session is based upon: [Secure the pipeline and CI/CD workflow](https://docs.microsoft.com/azure/cloud-adoption-framework/secure/best-practices/secure-devops).
+
+Guidance specifically for [Azure pipelines](https://docs.microsoft.com/azure/devops/pipelines/security/overview?view=azure-devops), which describes the different components you can secure.
+
+For workflows hosted on GitHub, there is guidance on [Security hardening](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions). For authenticating your GitHub workflows against Azure Active Directory, there is a (currently preview!) way of authenticating based on [workload identity](https://docs.microsoft.com/azure/active-directory/develop/workload-identity-federation). Guidance on how to configure and use this can be found in the [GitHub docs on Configuring OpenID Connect in Azure](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-azure) and in the [Microsoft docs on Use GitHub Actions to connect to Azure](https://docs.microsoft.com/azure/developer/github/connect-from-azure?tabs=azure-portal%2Cwindows).
+
+![devsecops-controls](./media/devsecops-controls.png)
```

Comment:
All other pages have this one on top. Would prefer to move it to the top here as well.

Reply:
Done

Reply:
@vermegi can you please approve for merge.